### DISCUSSION TITLE
Getting the total sum for each nested array group

### DISCUSSION THREAD

**Discussion by oliseh6123** [react-hook-form/react-hook-form#3176] [Q&A]
lets say i have a nested array (which is the child) in  another array (which is the parent), i want to be able add the group of children input value and get the total in each respective parent
append[0]child[0] field = 5
append[0]child[1] field = 3
parent[0]total field= 8

append[1]child[0] field = 11
append[1]child[1] field = 5
parent[1]total field= 16

**Comment by amsimon** [comment]
Did you find a solution to this?
